Formula/rgbds.rb

```diff
@@ -14,8 +14,11 @@ class Rgbds < Formula
     sha256 "c08b856ecf4cb57390ec99241d0bc87ba623536ab7e3e11d5cc23334230ff1cd" => :mavericks
   end
 
+  depends_on "pkg-config" => :build
```

Comment:
@woodruffw I get `/usr/local/opt/libpng/include/libpng16` (system) with the Formula["libpng"] method (which I am not familiarized at), and `/usr/local/Cellar/libpng/1.6.26/include/libpng16` (brew) with `pkg-config`. I feel like the brew'd version is desired over the system one, but I am unsure. What do you think?